### PR TITLE
Minor improvements to TxBuilder and Engine

### PR DESCRIPTION
**Pull request by Geod24**
I was looking for the bug of the day when I noticed there was some room for improvement here.

### CODE REVIEW

File: source/agora/script/Engine.d [hunk 1]
```diff
@@ -1008,14 +1008,14 @@ public class Engine
...
-        if (key_bytes.length != Point.sizeof)
+        if (key_bytes.length != PublicKey.sizeof)
         {
             static immutable err3 = op.to!string
                 ~ " opcode requires 32-byte public key on the stack";
             return err3;
         }
 
-        const Point point = Point(key_bytes);
+        const PublicKey point = PublicKey(key_bytes);
```
Comment (linked0) [inline source/agora/script/Engine.d]:
```suggestion
        const PublicKey key = PublicKey(key_bytes);
```

**Review by linked0 — APPROVED** [review]
LGTM one nit

File: source/agora/script/Engine.d (hunk 1 above)
Comment (Geod24) [reply source/agora/script/Engine.d]: Went with something slightly different, but incorporated the name change

**Review by hewison-chris — APPROVED** [review]
LGTM

**Review by kchulj — APPROVED** [review]
LGTM

**Pull request merged**